### ISSUE TITLE
[1.16.5] Mob's combat config is not working

### ISSUE THREAD

**Issue by Lemonmito** [BobMowzie/MowziesMobs#578]

**Comment by BobMowzie** [comment]
Did you spawn new mobs after changing the config? It doesn’t work on old ones even if you reload.

**Comment by Lemonmito** [comment]
yes, i create a new world and spawn new mobs after changing the config, it still keep the default health value.
![QQ截图20220130185809](https://user-images.githubusercontent.com/75903896/151696909-710a8ac6-0d1c-4227-902b-bc0570fda9ba.png)
![2022-01-30_18 57 57](https://user-images.githubusercontent.com/75903896/151696911-b44d5957-dceb-4a8d-a487-906b027e5d41.png)

**Comment by BobMowzie** [comment]
Did you restart your game?
How are you editing the config file in-game? As far as I know, Forge removed that feature.

**Comment by Lemonmito** [comment]
> Did you restart your game? How are you editing the config file in-game? As far as I know, Forge removed that feature.

Yes, I also tried to modify the config file directly, but it still didn't work.
By using two mods, "Configured" and "Game Menu Mod Options".
(btw. I have tried to modify the 1.5.19 version and there is no problem.)

**Comment by Terra-B-Welch** [comment]
can confirm, tried to give ferrous wroughtnaut a 3x health and 2x damage mod, neither applied.
